PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1545,6 +1551,11 @@ bool Notepad_plus::replaceInFiles()
 
 	progress.close();
 
+	// Show large ignored files which could not be opened for find/repace in folder
+	// It is just to intimate user about the ingored file details
+	if (isIgnoreLargeFileMessage && ignoredFiles.size() > 0)

```

Comment:
!ignoredFiles.empty()

Reply:
I feel it should not matter. Because 
1. If we use size then number of operations = 2 (one for calculating size + one for comparison again 0).
2. If we use empty then number of operations = 2 (one for comparing last==first + one for ! )

![image](https://user-images.githubusercontent.com/14791461/35769919-bf506b40-0938-11e8-84e8-6d13e2b2a35a.png)

Reply:
I agree they both are equally performant, however you're talking about implementation details where I noticed the style. `empty()` just expresses your intent better.

Besides, even though we both know `size()` and `empty()` are always going to stay the same, it's not guaranteed by the standard. Standard only guarantees that both methods must be of constant complexity for `std::vector`.